test/system/libs/helpers.bash

```diff
@@ -577,3 +577,56 @@ function is_fedora_rawhide() (
 
   return 0
 )
+
+
+# Creates a container with org.freedesktop.Flatpak.SessionHelper D-Bus interface
+#
+# Pulling of an image is taken care of by the function
+#
+# Parameters:
+# ===========
+# - container_name - name of the container
+function create_container_flatpak_session_helper() (
+  local container_name="$1"
+
+  local default_image
+  default_image="$(get_default_image)"
+
+  pull_default_image
+
+  gdbus call \
+    --session \
+    --dest org.freedesktop.Flatpak \
+    --object-path /org/freedesktop/Flatpak/SessionHelper \
+    --method org.freedesktop.Flatpak.SessionHelper.RequestSession
+
+  podman_cmd=(
+      podman create \
+          --dns none \
+          --env TOOLBOX_PATH="$TOOLBX" \
+          --name "$container_name" \
+          --network host \
+          --no-hosts \
+          --pid host \
+          --privileged \
+          --userns=keep-id \
+          --user root:root \
+          --volume "$HOME":"$HOME":rslave \
+          --volume "$TOOLBX":/usr/bin/toolbox \
+          --volume "$XDG_RUNTIME_DIR":"$XDG_RUNTIME_DIR" \
+          --volume "$XDG_RUNTIME_DIR/.flatpak-helper/monitor":/run/host/monitor \
+          --volume /etc:/run/host/etc \
+          --volume /usr:/run/host/usr:rslave \
+          "$default_image" \
+          toolbox init-container \
+                  --home "$HOME" \
+                  --shell "$SHELL" \
```

Comment:
Let's add the `--monitor-host` option to `init-container` here.

We [deprecated --monitor-host](https://github.com/containers/toolbox/commit/58638c594004adb8a093ca1c774c082a37011282) and stopped using it from Toolbx 0.0.99.5.  It means that containers that were created by Toolbx < 0.0.97 to use `org.freedesktop.Flatpak.SessionHelper` still have the option, and `init-container` should continue to handle it.  It is hard or rare to test deprecated code paths, so using it in this test is a good opportunity to test it.